### ISSUE TITLE
Support for matching multiple classes/files by name in config

### ISSUE THREAD

**Issue by JanMikes** [rectorphp/rector#1003]
Before:
```yaml
services:
    Rector\Rector\Typehint\ParentTypehintedArgumentRector:
        Nette\Application\IPresenter: # I want to select multiple classes here
            run:
                $appRequest: Symfony\Component\HttpFoundation\Request
```

After:
```yaml
services:
    Rector\Rector\Typehint\ParentTypehintedArgumentRector:
        *Presenter:  # this will match all classes ending to 'Presenter' word
            run:
                $appRequest: Symfony\Component\HttpFoundation\Request
```

**Comment by JanMikes** [comment]
@TomasVotruba looking forward this one :+1:

**Comment by TomasVotruba** [comment]
Resolved by #1004
